### ISSUE TITLE
Release of next version of OMJulia?

### ISSUE THREAD

**Issue by CSchoel** [OpenModelica/OMJulia.jl#37]
I would really like to be able to install the current repository version of OMJulia with `pkg> add OMJulia`, especially because of the fix for #22. As I understand this only requires an update of the Project.toml and a merge request on https://github.com/JuliaRegistries/General .

Currently, I need to manually clone the repository and activate the local copy, because due to https://github.com/JuliaLang/julia/issues/33111 even `pkg> add https://github.com/OpenModelica/OMJulia.jl` is not working. This is quite cumbersome to include in a CI script. :weary:

**Comment by arun3688** [comment]
@CSchoel  you can use something like below, This should fetch the latest changes

```
julia> import Pkg
julia> Pkg.add(PackageSpec(url=”https://github.com/OpenModelica/OMJulia.jl”))
```

**Comment by CSchoel** [comment]
Thanks for the hint, but this gives me the same error as when I use `pkg> add https://github.com/OpenModelica/OMJulia.jl` (which probably is just the same as your example anyway, just in REPL syntax?):

```
ERROR: failed to clone from https://github.com/OpenModelica/OMJulia.jl, error: GitError(Code:ERROR, Class:Invalid, invalid version 0 on git_proxy_options)
```

From the discussion at https://github.com/JuliaLang/julia/issues/33111 I would hope that this issue does not affect packages that are officially registered, since they do not have to be cloned internally.

**Comment by sjoelund** [comment]
You can also use simply `add OMJulia#master`. But it's probably good to make more releases anyway, Arun :)

**Comment by AnHeuermann** [comment]
I tagged v0.2.0 and v0.2.1 and will create a new release v0.2.2 with the updated LICESNE on the master branch (fd2e3a1d16fa943e731616003fb85ed8733af6b9).

Then I'll try to register the package in the Julia registry.

**Comment by AnHeuermann** [comment]
We should probably setup [Registrator.jl](https://github.com/JuliaRegistries/Registrator.jl) and [TagBot](https://github.com/JuliaRegistries/TagBot) to automate this for the future.

**Comment by AnHeuermann** [comment]
v0.2.0 is now registered: https://github.com/JuliaRegistries/General/pull/78188

I'll register bug fix releases v0.2.1/v0.2.2 as well.

**Comment by AnHeuermann** [comment]
v0.2.1 is now released and represents the newest state of OMJulia:
https://github.com/OpenModelica/OMJulia.jl/releases/tag/v0.2.1
